Apps/Playground/Win32/App.cpp

```diff
@@ -131,6 +131,8 @@ namespace
 
             Babylon::Polyfills::Console::Initialize(env, [](const char* message, auto) {
                 OutputDebugStringA(message);
+                printf("%s", message);
+                fflush(stdout);
```

Comment:
This is needed because of the merge with validation tests? Was the CI missing output then? cc @CedricGuillemet

Reply:
Ya, the validation tests for this change should fail for this PR until the Babylon.js change gets deployed. In the first CI runs for this change, the validation test process hung as expected but was not considered to be failed. Cedric made some changes to fix that and added them to this PR.